frontend/src/utils/AxiosApis.ts

```diff
@@ -0,0 +1,21 @@
+import axios from "axios";
+
+const AxiosReportersApi = axios.create({
+  baseURL: "https://jsonplaceholder.typicode.com/",
+  timeout: 1000,
+  headers: { "X-Custom-Header": "foobar" },
+});
+
+const AxiosUserApi = axios.create({
+  baseURL: "http://localhost:5050/api/user/",
+  timeout: 1000,
+  headers: { "X-Custom-Header": "foobar" },
+});
+
+const AxiosComplaintApi = axios.create({
+  baseURL: "http://localhost:5050/api/complaint/",
```

Comment:
matter of taste (but up to you):
I would actually make only one extra instance for the back-end `http://localhost:5050/api/`
and you use `user` or `complaint` where you need it. This way you can still see clear what endpoint is being triggered.